config.js.example

```diff
@@ -9,15 +9,15 @@ const config = {
   "support": [],
 
   // Your Bot's Token. Available on https://discordapp.com/developers/applications/me
-  "token": "mfa.VkO_2G4Qv3T--NO--lWetW_tjND--TOKEN--QFTm6YGtzq9PH--4U--tG0",
+  "token": "Mzg5MDExNjQxNjI0MTY2NDAy.DRRjLQ.QXEaeJKfLHq-HhH2pBreBojBtws",
```

Comment:
You exposed your bot token, all you needed to do was copy/paste the original example from the repo.

Reply:
You submitted the edited config again lol